src/community/jwt-headers/jwt-headers-gs/src/main/resources/applicationContext.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ (c) 2024 Open Source Geospatial Foundation - all rights reserved
+  ~ This code is licensed under the GPL 2.0 license, available at the root
+  ~ application directory.
+  ~
+  -->
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:sec="http://www.springframework.org/schema/security"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans
+          http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+          http://www.springframework.org/schema/security
+          http://www.springframework.org/schema/security/spring-security-3.0.4.xsd">
+
+	<bean class="org.geoserver.platform.ModuleStatusImpl">
+		<constructor-arg index="0" value="jwt-headers" />
+		<constructor-arg index="1" value="GeoServer JWT Headers Security" />
+	</bean>
+
+	<!-- enables Keycloak as an auth-provider -->
```

Comment:
should this be talking about keycloak?

Reply:
I copied it from the keycloak module - changed!